tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/client-logo-block.html

```diff
@@ -1,31 +0,0 @@
-{% load wagtailimages_tags wagtailcore_tags %}
-{# `value` is a ListBlock of ImageWithLinkBlock items #}
-{% if value or quotes %}
-
-    {% if title %}
-        {% include "patterns/atoms/section-title/section-title.html" with title=title %}
-    {% endif %}
-
-    {% if value %}
-        <ul>
-            {% for client in value %}
-                <li>
-                    {% with link=client.link.0.value %}
-                        {% if link %}
-                            <a href="{{ link.url }}" title="{{ link.text }}">
-                                {% image client.image width-400 loading="lazy" alt="" %}
-                                <span aria-hidden="true">{{ link.text }}</span>
-                            </a>
-                        {% else %}
-                            {% image client.image width-400 loading="lazy" alt="" %}
-                        {% endif %}
-                    {% endwith %}
-                </li>
-            {% endfor %}
-        </ul>
-    {% endif %}
-
-    {% if quotes %}
-        {% include "patterns/molecules/quote-slider/quote-slider.html" with items=quotes %}
```

Comment:
We can also remove `patterns/molecules/quote-slider/quote-slider.html` and `patterns/atoms/quote-slide/quote-slide.html`, as they are no longer called anywhere in the codebase after your changes.

Reply:
You might also need to update `streamfield/streamfield-example-work.html` to remove any similar code to the templates we are removing.